examples/cache-with-aws/README.md

```diff
@@ -0,0 +1,30 @@
+# Java Client SDK
+
+_Read this in other languages_: [日本語](README.ja.md)
+
+<br>
+
+## Running the Examples
+
+- You do not need gradle to be installed
+- JDK 14 or above is required to run the example
+- A Momento Auth Token is required, you can generate one using the [Momento CLI](https://github.com/momentohq/momento-cli)
```

Comment:
Wherever you see this in our READMEs now, we need to update it. The latest CLI doesn't support generating tokens anymore, we should direct users to the console. The text in the main README in this repo is up to date.

Reply:
Will do!